### PR TITLE
Completed css-pseudo-classes exercise

### PR DESCRIPTION
**Pull request by wang-morris**
https://user-images.githubusercontent.com/103090771/168950394-f460c41d-82ca-47ae-a4d0-d66c6a090633.mp4

### CODE REVIEW

File: css-pseudo-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Pseudo Classes</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h1>Hover Button</h1>
+  <input type="button" name="large-button" value="I'm a button!">
```
Comment (BrettAlbright) [inline css-pseudo-classes/index.html]: this is not a form, so you want to use a button here.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
one small update!

**Comment by wang-morris** [comment]
https://user-images.githubusercontent.com/103090771/168951179-2f514ba3-50dc-4646-9d5a-dc0dfbf129dd.mp4

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**